data.sql

```diff
@@ -10,35 +10,68 @@
--- Set the owner_id value based on the animal's owner
-UPDATE animals
-SET owner_id = (SELECT id FROM owners WHERE full_name = CASE name
-                                                            WHEN 'Agumon' THEN 'Sam Smith'
-                                                            WHEN 'Gabumon' THEN 'Jennifer Orwell'
-                                                            WHEN 'Pikachu' THEN 'Jennifer Orwell'
-                                                            WHEN 'Devimon' THEN 'Bob'
-                                                            WHEN 'Plantmon' THEN 'Bob'
-                                                            WHEN 'Charmander' THEN 'Melody Pond'
-                                                            WHEN 'Squirtle' THEN 'Melody Pond'
-                                                            WHEN 'Blossom' THEN 'Melody Pond'
-                                                            WHEN 'Angemon' THEN 'Dean Winchester'
-                                                            WHEN 'Boarmon' THEN 'Dean Winchester'
-                                                       END);
+-- INSERT INTO owners (full_name, age)
+-- VALUES ('Sam Smith', 34),
+--        ('Jennifer Orwell', 19),
+--        ('Bob', 45),
+--        ('Melody Pond', 77),
+--        ('Dean Winchester', 14),
+--        ('Jodie Whittaker', 38);
+
+--        -- Set the species_id value based on the animal's name
+-- UPDATE animals
+-- SET species_id = (SELECT id FROM species WHERE name = CASE WHEN name LIKE '%mon' THEN 'Digimon' ELSE 'Pokemon' END);
+
+-- -- Add the foreign key constraint to the species_id column
+-- ALTER TABLE animals ADD CONSTRAINT fk_species FOREIGN KEY (species_id) REFERENCES species(id);
+-- -- Set the owner_id value based on the animal's owner
+-- UPDATE animals
+-- SET owner_id = (SELECT id FROM owners WHERE full_name = CASE name
+--                                                             WHEN 'Agumon' THEN 'Sam Smith'
+--                                                             WHEN 'Gabumon' THEN 'Jennifer Orwell'
+--                                                             WHEN 'Pikachu' THEN 'Jennifer Orwell'
+--                                                             WHEN 'Devimon' THEN 'Bob'
+--                                                             WHEN 'Plantmon' THEN 'Bob'
+--                                                             WHEN 'Charmander' THEN 'Melody Pond'
+--                                                             WHEN 'Squirtle' THEN 'Melody Pond'
+--                                                             WHEN 'Blossom' THEN 'Melody Pond'
+--                                                             WHEN 'Angemon' THEN 'Dean Winchester'
+--                                                             WHEN 'Boarmon' THEN 'Dean Winchester'
+--                                                        END);
```

Comment:
- [ ] Please remove unused code so that your code base is clean and uncluttered.